event_question_descr_block/models/event_question_answer.py

```diff
@@ -0,0 +1,38 @@
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class EventQuestionAnswer(models.Model):
+    _inherit = "event.question.answer"
+
+    is_blocking = fields.Boolean(string="Is Blocking?")
+
+
+class EventRegistrationAnswer(models.Model):
+    _inherit = "event.registration.answer"
+
+    def check_answer_blocking(self, answer_id):
+        answer = self.env["event.question.answer"].browse(answer_id)
+        if answer and answer.is_blocking:
+            raise UserError(
+                _(" Based on your Answers you cannot register for this event.")
+            )
```

Comment:
This is a constraint check.
Can we use an api.constrains?

Reply:
@dreispt Yep updated the code. Please review.